src/core/tests/eval.cpp

```diff
@@ -434,7 +434,7 @@ TEST(eval, evaluate_broadcast_v1_numpy_dyn) {
     ASSERT_EQ(result_val, expec);
 }
 
-TEST(eval, evaluate_broadcast_v1_pdpd) {
+TEST(eval, evaluate_broadcast_v1_paddle) {
```

Comment:
I suppose such places should be kept as is, because in enum we name `PDPD`

Reply:
fixed